var/spack/repos/builtin/packages/py-easybuild-easyblocks/package.py

```diff
@@ -16,10 +16,11 @@ class PyEasybuildEasyblocks(PythonPackage):
     maintainers("boegel")
 
     version("4.0.0", sha256="a0fdef6c33c786e323bde1b28bab942fd8e535c26842877d705e692e85b31b07")
+    version("4.7.0", sha256="c23e81cbaa3e4fa5ab1bb8b2db759332867d61110bf4ec34763ea170780f0655")
```

Comment:
Can you sort these newest to oldest?

Reply:
Is this new? I thought we always sorted oldest to newest?

Reply:
Uh oh.  I didn't check this change on your other PRs.

Reply:
No, this is not new.  Spack prefers newer versions be listed first.

Reply:
Oh I just did this backwards for the most recent dependencies I added. Sorry lol I was confused because I thought @adamjstewart was suggesting I put the most recent versions at the bottom not the top.

Reply:
Done! @tldahlgren the rest of the PRs I did should be good. I believe this was a one off this morning when I wasn't paying attention.